src/new.rs

```diff
@@ -89,13 +98,24 @@ pub fn create_new_project<P: AsRef<Path>>(dest: P) -> Result<()> {
     Ok(())
 }
 
-pub fn create_new_document(doc_type: &str, name: &str, config: &Config) -> Result<()> {
+pub fn create_post(name: &str, config: &Config) -> Result<()> {
     let path = Path::new(&config.source);
-    let full_path = &path.join(&config.posts).join(name);
+
+    let local = Local::now().format("%d %b %Y %H:%M:%S %z").to_string();
+    let vector: Vec<u8> = Vec::from(new_post_tmpl);
+    let mut out = String::from_utf8(vector).unwrap();
+    out = out.replace("%title%", name);
+    out = out.replace("%date%", &local);
+    let new_post = &path.join(format!("{}/{}.md", &config.posts, name));
```

Comment:
I can see either direction.  We just need to decide and document the behavior and why we made the choices we did in the PR.  We can later transfer the behavior documentation over to the user docs.

So if its the title
- Should we change [`FILENAME`](https://github.com/cobalt-org/cobalt.rs/blob/master/src/main.rs#L111) to `TITLE`?
- Should we slugify the name for creating the file name?  I'd lean towards "yes"
- Should the user be expected to quote the parameter or should we capture all following text as part of the name (not sure if multiple occurences or multiple values is the right approach in clap)?